Xamarin.Forms.Core/IDispatcher.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Xamarin.Forms
+{
+	public interface IDispatcher
+	{ 
+		void BeginInvokeOnMainThread(Action action);
+		bool IsInvokeRequired();
```

Comment:
I think it's better to make `IsInvokeRequired` as property with a getter.

Reply:
Changed!